test/test_managercli.py

```diff
@@ -278,7 +278,7 @@ def test_main_long_help(self):
     def test_cli_in_container_error_message(self, mock_in_container):
         with patch.object(sys, 'argv', ['subscription-manager', 'version']):
             mock_in_container.return_value = True
-            err_msg = 'subscription-manager is disabled when running inside a container.'\
+            err_msg = 'subscription-manager is operating in container mode.'\
                       ' Please refer to your host system for subscription management.\n\n'
```

Comment:
this will need to be updated to the string change mentioned in my previous comment